cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -377,6 +378,12 @@ elseif(RISCV)
   set(CPU_DISPATCH "RVV" CACHE STRING "${HELP_CPU_DISPATCH}")
   set(CPU_BASELINE "RVV" CACHE STRING "${HELP_CPU_BASELINE}")
 
+elseif(LOONGARCH64)
+  ocv_update(CPU_LASX_TEST_FILE "${OpenCV_SOURCE_DIR}/cmake/checks/cpu_lasx.cpp")
+  ocv_update(CPU_KNOWN_OPTIMIZATIONS "LASX")
+  ocv_update(CPU_LASX_FLAGS_ON "-mlasx")
+  set(CPU_BASELINE "DETECT" CACHE STRING "${HELP_CPU_BASELINE}")
```

Comment:
Yes, I noticed it. The reason of checking failure is no -mlasx flag. May be there is a bug in DETECT mode. The check is done as follow:
`ocv_check_compiler_flag(CXX "${CPU_BASELINE_FLAGS}" "${_varname}" "${CPU_${OPT}_TEST_FILE}")`,
I think `CPU_BASELINE_FLAGS` should be `CPU_${OPT}_FLAGS_ON`, right ?

Reply:
`DETECT` is used for detection of current compiler flags (without any extra flags, but it could properly handle `-march=native` case).
`DETECT` is similar to empty (default) baseline.

To learn more how it works please take a look on x86 part.
Also there is Wiki page about that: https://github.com/opencv/opencv/wiki/CPU-optimizations-build-options

Reply:
Ok, I get it.  
LASX can be set by default. I will config it.